astro/src/content/blog/test-saml-configuration.mdx

```diff
@@ -0,0 +1,115 @@
+</Aside>
+
+## Test FusionAuth SAML Configuration With IAMShowcase
+There are generic [instructions](https://sptest.iamshowcase.com/instructions#start) on the IAMShowcase site, but the rest of this tutorial describes how to set up FusionAuth as a SAML IdP.
+
+You will need an instance of [FusionAuth up and running](/docs/quickstarts/5-minute-docker).
+
+### Create A New Application Named IAMShowcase Example
+
+Log into FusionAuth and select <Breadcrumb>Applications</Breadcrumb>. Give the application a <InlineField>name</InlineField> such as `IAMShowcase Example`. If you have more than one tenant, select the default <InlineField>Tenant</InlineField>. Then click the blue disk to save the configuration.
+
+### Retrieve The Configuration Values From The IAMShowcase Website
+
+The IAMShowcase site provides a metadata file that some IdPs can import. You can get the XML from the IAMShowcase site by clicking on [Get Metadata](https://sptest.iamshowcase.com/instructions#start). FusionAuth does not support importing of the metadata file. To see the values used, view the XML and manually enter the information in the SAML tab of the application in FusionAuth.
+
+Example of metadata provided by IAMShowcase:
+```
+<md:EntityDescriptor xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata" xmlns:ds="http://www.w3.org/2000/09/xmldsig#" entityID="IAMShowcase" validUntil="2025-12-09T09:13:31.006Z">
+<md:SPSSODescriptor AuthnRequestsSigned="false" WantAssertionsSigned="true" protocolSupportEnumeration="urn:oasis:names:tc:SAML:2.0:protocol">
+<md:NameIDFormat>urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified</md:NameIDFormat>
+<md:NameIDFormat>urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress</md:NameIDFormat>
+<md:AssertionConsumerService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST" Location="https://sptest.iamshowcase.com/acs" index="0" isDefault="true"/>
+</md:SPSSODescriptor>
+</md:EntityDescriptor>
+```
+
+### Configure The IAMShowcase Example Application For SAML
+
+Select <Breadcrumb>Applications</Breadcrumb> and <InlineUIElement>Edit</InlineUIElement> the IAMShowcase Example application in FusionAuth. Select the <Breadcrumb>SAML</Breadcrumb> tab of the application and configure it with the following values.
+
+<InlineField>Enabled</InlineField> `True`
+
+<InlineField>Issuer</InlineField> `IAMShowcase`
+This is the entityID from the EntityDescriptor node in the matadata provided by IAMShowcase.
+```
+<md:EntityDescriptor xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata" xmlns:ds="http://www.w3.org/2000/09/xmldsig#" entityID="IAMShowcase" validUntil="2025-12-09T09:13:31.006Z">
+```
+
+<InlineField>Authorized Redirect URLs</InlineField> `https://sptest.iamshowcase.com/acs`
+This is the Location from the AssertionConsumerService node in the matadata provided by IAMShowcase.
```

Comment:
```suggestion
This is the `Location` from the `AssertionConsumerService` node in the metadata provided by IAMShowcase. Here's the relevant excerpt from the example XML above.
```

Reply:
also makes sense to spellcheck everything (would have caught `matadata`. I like to copy the blog text from the local astro instance and paste into google docs, which also catches tense errors.